charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 19.2.2
+version: 19.2.3
```

Comment:
apologies for the delayed response. Can you please fix the conflicts? I will approve the PR.